packages/viewer/src/router/storeSync/params/featureInfo.param.ts

```diff
@@ -0,0 +1,50 @@
+import type { RouteLocationNormalizedGeneric } from 'vue-router'
+
+import UrlParamConfig, {
+    STORE_DISPATCHER_ROUTER_PLUGIN,
+} from '@/router/storeSync/UrlParamConfig.class'
+import { getDefaultValidationResponse } from '@/router/storeSync/validation'
+import useUIStore, { FeatureInfoPositions, UIStoreActions } from '@/store/modules/ui.store'
+import { isEnumValue } from '@/utils/utils'
+
+function parseFeatureInfoPosition(urlParamValue?: string): FeatureInfoPositions | undefined {
+    if (!urlParamValue) {
+        return undefined
+    }
+    if (isEnumValue<FeatureInfoPositions>(FeatureInfoPositions.DEFAULT, urlParamValue)) {
+        return FeatureInfoPositions.DEFAULT
+    }
+    if (isEnumValue<FeatureInfoPositions>(FeatureInfoPositions.TOOLTIP, urlParamValue)) {
+        return FeatureInfoPositions.TOOLTIP
+    }
+    if (isEnumValue<FeatureInfoPositions>(FeatureInfoPositions.BOTTOMPANEL, urlParamValue)) {
+        return FeatureInfoPositions.BOTTOMPANEL
+    }
+    if (isEnumValue<FeatureInfoPositions>(FeatureInfoPositions.NONE, urlParamValue)) {
+        return FeatureInfoPositions.NONE
+    }
```

Comment:
Wouldn't it be more readable to have it in a way like this: 
```
 function parseFeatureInfoPosition(urlParamValue?: string): FeatureInfoPositions | undefined {
    if (Object.values(FeatureInfoPositions).includes(urlParamValue)){
        return urlParamValue as FeatureInfoPositions
    }
    return undefined
  }
```
This has the same behavior, and is a bit more readable in my opinion. (I don't think we would ever have another option for the tooltip position, or that we would remove some options, but being able to modify the enum and not have to touch this function is a small plus too )

Reply:
You can't do Object.values on TS enum type, or what we could do is decide not to use TS enum, and stick with object constants

(you can, but the value you give to the includes, here `urlParamValue`, will not be recognized as a value of, because it's a string)